src/overlays/actors/ovl_En_Al/z_en_al.c

```diff
@@ -18,100 +19,895 @@ void EnAl_Draw(Actor* thisx, GlobalContext* globalCtx);
+        this->unk_4C2 |= 0x200;
+        this->actor.child = NULL;
+        this->unk_4E2 = 20;
+        this->unk_364 = 0;
+        this->actionFunc = func_80BDF5E8;
+    } else {
+        Math_ApproachS(&this->actor.shape.rot.y, this->actor.world.rot.y, 3, 0x2AA8);
+    }
+}
+
+void EnAl_Init(Actor* thisx, GlobalContext* globalCtx) {
+    EnAl* this = THIS;
+
+    ActorShape_Init(&this->actor.shape, 0.0f, NULL, 0.0f);
+    SkelAnime_InitFlex(globalCtx, &this->skelAnime, &object_al_Skel_00A0D8, NULL, this->jointTable, this->morphTabke,
+                       27);
+    this->unk_4F8 = -1;
+    func_80BDE27C(this, 1);
+    Collider_InitAndSetCylinder(globalCtx, &this->unk_310, &this->actor, &sCylinderInit);
+    CollisionCheck_SetInfo2(&this->actor.colChkInfo, DamageTable_Get(0x16), &sColChkInfoInit);
+    Actor_SetScale(&this->actor, 0.01f);
+    this->unk_35C = 0;
+    this->actionFunc = func_80BDF5E8;
+
+    this->actionFunc(this, globalCtx);
+}
+
+void EnAl_Destroy(Actor* thisx, GlobalContext* globalCtx) {
+    EnAl* this = THIS;
+
+    Collider_DestroyCylinder(globalCtx, &this->unk_310);
+}
+
+void EnAl_Update(Actor* thisx, GlobalContext* globalCtx) {
+    EnAl* this = THIS;
+
+    func_80BDEC2C(this, globalCtx);
+
+    this->actionFunc(this, globalCtx);
+
+    func_80BDF064(this, globalCtx);
+
+    if (this->unk_35C != 0) {
+        func_80BDE250(this);
+        func_80BDEE5C(this);
+        func_8013C964(&this->actor, globalCtx, this->unk_4D4, 30.0f, this->unk_4F0, this->unk_4C2 & 7);
+        func_80BDE318(this, globalCtx);
+    }
+}
+
+s32 EnAl_OverrideLimbDraw(GlobalContext* globalCtx, s32 limbIndex, Gfx** dList, Vec3f* pos, Vec3s* rot, Actor* thisx) {
+    switch (limbIndex) {
+        case 3:
+        case 11:
+        case 12:
+        case 13:
+        case 14:
+        case 15:
+            *dList = NULL;
+            break;
+
+        case 16:
+            break;
+    }
+    return false;
+}
+
+void EnAl_PostLimbDraw(GlobalContext* globalCtx, s32 limbIndex, Gfx** dList, Vec3s* rot, Actor* thisx) {
+    EnAl* this = THIS;
+
+    switch (limbIndex) {
+        case 3:
+            Matrix_CopyCurrentState(&this->unk_190[0]);
+            break;
+
+        case 11:
+            Matrix_CopyCurrentState(&this->unk_190[1]);
+            break;
+
+        case 12:
+            Matrix_CopyCurrentState(&this->unk_190[2]);
+            break;
+
+        case 13:
+            Matrix_CopyCurrentState(&this->unk_190[3]);
+            break;
+
+        case 14:
+            Matrix_CopyCurrentState(&this->unk_190[4]);
+            break;
+
+        case 15:
+            Matrix_CopyCurrentState(&this->unk_190[5]);
+            break;
+
+        case 16:
+            Matrix_MultiplyVector3fByState(&D_80BE0070, &this->actor.focus.pos);
+            Math_Vec3s_Copy(&this->actor.focus.rot, &this->actor.world.rot);
+            break;
+    }
+}
+
+void EnAl_TransformDraw(GlobalContext* globalCtx, s32 limbIndex, Actor* thisx) {
```

Comment:
```suggestion
void EnAl_TransformLimbDraw(GlobalContext* globalCtx, s32 limbIndex, Actor* thisx) {
```
Would prefer including Limb to match the other limb draws (override and post).

Reply:
I don't remember saying `TransformDraw` by itself, but if I did, that would have been my mistake. My bad.